pep-0681.rst

```diff
@@ -218,6 +219,13 @@ customization of default behaviors:
   assumed to be True or False if it is omitted by the caller. If not
   specified, ``kw_only_default`` will default to False (the default
   assumption for dataclass).
+* ``transform_descriptor_types`` indicates whether, for fields whose
+  annotated types are descriptors, the corresponding parameter in the
+  synthesized ``__init__`` method should be the descriptor's
+  ``__set__`` function's value parameter type. If False, the
```

Comment:
This sentence is really dense (4 lines long!). Please may you rephrase slightly / break to multiple sentences?

A

Reply:
How about this? Slight change to the second sentence of @JelleZijlstra's suggestion to make it clear that this affects all such `__init__` parameters:

```suggestion
* ``transform_descriptor_types`` affects fields annotated with
  descriptor types that define a ``__set__`` method. If True, the type
  of each parameter corresponding to such a field on the synthesized
  ``__init__`` method will be the type of the value parameter to the
  descriptor's ``__set__`` method. If False, the
```

Reply:
I like it! Maybe switch "on the synthesized `__init__` method" and "corresponding to such a field"; it reads more naturally to me that way.